frontend/public/components/edit-yaml.jsx

```diff
@@ -278,47 +305,69 @@ export const EditYAML_ = connect(stateToProps)(
           });
         }
 
-        save() {
-          const { onSave, t } = this.props;
-          const { owner } = this.state;
-          let obj;
+        performDryRun(objs) {
+          const { models } = this.props;
+          let dryRunResources = [];
+          this.setState({ success: null, errors: null, sending: true }, () => {
+            dryRunResources = createResources(objs, models, true);
+            //catch these individually so we can report out all errors
+            dryRunResources.forEach((request, i) =>
+              request.catch((error) => this.handleErrors(objs[i], error?.message)),
+            );
+            Promise.all(dryRunResources)
+              .then(() => {
+                this.setState({
+                  errors: null,
+                  sending: false,
+                  resourceObjects: objs,
+                });
+                this.showResults(true);
+              })
+              .catch(() => {
+                //catch this error but do nothing since we show individual errors above
+              });
+          });
+        }
 
-          if (onSave) {
-            onSave(this.getEditor().getValue());
-            return;
-          }
+        showResults(value) {
```

Comment:
Naming nit: I'd call this `setDisplayResults` so it's clear the boolean parameter is setting that value. Otherwise it looks like `showResults` always shows the results and the parameter is a flag that controls some other behavior. Also it's good to use a consistent name with the state value.